mentors/README.md

```diff
@@ -0,0 +1,76 @@
+## Become a Mentor
```

Comment:
Is this a new, incomplete file that's being created, or an old incomplete file that you're just moving?  I can't tell from the PR (too many files).

Reply:
If it's new, can you put a WIP header in it somewhere?

Reply:
Yes, if this is going to be merged into main and be on the "live" readme, then if it's unfinished content it should be clearly called out. 👍

Reply:
Updated in https://github.com/cncf/mentoring/pull/927